.github/workflows/build_dataops.yml

```diff
@@ -8,6 +8,9 @@ on:
 jobs:
   buildx-dataops:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        provider: ["bigquery"]
```

Comment:
Maybe we could also add general here?

Reply:
What is a "general" in this case? An image _without_ any tooling? We can easily create one but the question arises, if it would be a useful image at all (i.e., I moved `dbt-bigquery` to `requirements-bigquery.txt`, as it would not be useful for something like snowflake anyway).